test/unit/functional/collapse-and-batch.spec.js

```diff
@@ -238,7 +236,7 @@ describe('Collapse and Batch', function() {
             get([['lists', [0, 1], 'summary']]);
         var count = 0;
         obs.
-            doAction(function(res) {
+            subscribe(function(res) {
```

Comment:
the pattern of `doAction(() => expect(...)).subscribe(noop, done, done)` ensures errors thrown by `expect()` get forwarded to the `done` handler. If expectations happen in the subscribe handlers, the `done` handler never gets invoked.